src/rust/engine/src/scheduler.rs

```diff
@@ -40,6 +40,7 @@ struct InnerSession {
   // A place to store info about workunits in rust part
   workunit_store: WorkUnitStore,
   build_id: String,
+  workunit_reporting: bool,
```

Comment:
`should_record_workunits` would also be more consistent here.

Reply:
Changed